src/components/productDetails/card.jsx

```diff
@@ -38,13 +39,21 @@ const Card = ({
       </div>
       <div className="card-image actionButtonDiv">
         <button
-          className="button is-small actionButton"
+          className={`button is-small actionButton product-${product.id} ${listName}`}
           value={listName}
           onClick={e => {
             if (listName === "Related") {
               setShowModal(true);
+              recordInteraction(
+                `button.button.is-small.actionButton.product-${product.id}.${listName}`,
+                "related-items-comparison"
+              );
             } else {
               onClickButton(listName, product.id);
+              recordInteraction(
+                `button.button.is-small.actionButton.product-${product.id}.${listName}`,
+                "related-items-comparison"
```

Comment:
We talked about level of specificity, making sure we all have at least e.g. button & id